Octokit/Models/Request/SearchIssuesRequest.cs

```diff
@@ -337,4 +336,53 @@ public enum IssueTypeQualifier
         [Parameter(Value = "issue")]
         Issue
     }
+
+    public class RepositoryCollection : IEnumerable<string>
+    {
+        readonly ICollection<string> _repositories = new List<string>();
+
+        public IEnumerator<string> GetEnumerator()
+        {
+            return _repositories.GetEnumerator();
+        }
+
+        IEnumerator IEnumerable.GetEnumerator()
+        {
+            return GetEnumerator();
+        }
+
+        public void Add(string owner, string name)
```

Comment:
I added a commit with the single string overloads, you can just strip them out if you decide you don't want both.